modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminManagementToolbarDisplayContext.java

```diff
@@ -730,6 +783,8 @@ private long _getFileEntryTypeId() {
 	private List<DropdownItem> _getFilterNavigationDropdownItems() {
 		boolean extensionsIsEmpty = ArrayUtil.isEmpty(
 			_getExtensions(_httpServletRequest));
+		boolean tagsIsEmpty = SetUtil.isEmpty(
+			_getAssetTagIds(_httpServletRequest));
```

Comment:
Can you move this closer to its usage? If you can, I think it should be renamed to `active` (or inlined).

Reply:
Yes, I would inline it.

Reply:
this var will be used on posterior PRs as `extensionsIsEmpty` to know what the action is active. But I can inline it now, not problem.